lib/ansible/modules/monitoring/uptimerobot.py

```diff
@@ -28,7 +28,7 @@
             - Define whether or not the monitor should be running or paused.
         required: true
         default: null
-        choices: [ "started", "paused" ]
+        choices: [ "started", "paused", "new", "reset", "delete"]
```

Comment:
I would recommend to create a new module rather than modifying this. New module will support CRUD operations on given monitors, whereas this module will only manage state of monitors.